src/WaveTrack.cpp

```diff
@@ -1116,17 +1155,18 @@ void WaveTrack::HandleClear(double t0, double t1,
                   // Delete in the middle of the clip...we actually create two
                   // NEW clips out of the left and right halves...
 
-                  // left
-                  clipsToAdd.push_back
-                     ( std::make_unique<WaveClip>( *clip, mpFactory, true ) );
-                  clipsToAdd.back()->Clear(t0, clip->GetEndTime());
+                  auto leftClip = std::make_unique<WaveClip>(*clip, mpFactory, true);
+                  //i18n-hint Name assigned to the first clip on split
+                  leftClip->SetName(XC("%s 1", "clip name template").Format(leftClip->GetName()).Translation());
+                  leftClip->Clear(t0, clip->GetEndTime());
+                  clipsToAdd.push_back(std::move(leftClip));
 
-                  // right
-                  clipsToAdd.push_back
-                     ( std::make_unique<WaveClip>( *clip, mpFactory, true ) );
-                  WaveClip *const right = clipsToAdd.back().get();
-                  right->Clear(clip->GetStartTime(), t1);
-                  right->Offset(t1 - clip->GetStartTime());
+                  auto rightClip = std::make_unique<WaveClip>(*clip, mpFactory, true);
+                  //i18n-hint Name assigned to the second clip on split
+                  rightClip->SetName(XC("%s 2", "clip name template").Format(leftClip->GetName()).Translation());
```

Comment:
Does this test correctly?  I think this would crash because you use leftClip after moving from it.

Reply:
Typo, thanks!